addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -274,7 +275,8 @@ def _compute_from_holiday_type(self):
                 allocation.employee_ids = False
                 allocation.mode_company_id = False
             else:
-                allocation.employee_ids = self.env.context.get('default_employee_id') or self.env.user.employee_id
+                employee_ids = self.env.context.get('default_employee_id', self.env.user.employee_id)
+                allocation.employee_ids = [employee_ids] if isinstance(employee_ids, int) else employee_ids
```

Comment:
You can bypass the `isinstance` if you directly get the numeric ID of the current employee :-)

```suggestion
                allocation.employee_ids = [self.env.context.get('default_employee_id', self.env.user.employee_id.id)]
```

Reply:
Good call, thanks :-)